src/messages.h.jinja

```diff
@@ -0,0 +1,42 @@
+// NOTE: This file has been generated automatically by “update-message-registry.py”.
+//       Do not edit manually!
+
+#ifndef _XKBCOMMON_MESSAGE_H_
```

Comment:
We use this format for public headers; for internal headers should use `MESSAGES_H`

Reply:
Fixed.